src/music_gui.cpp

```diff
@@ -799,6 +804,5 @@ static WindowDesc _music_window_desc(
 
 void ShowMusicWindow()
 {
-	if (BaseMusic::GetUsedSet()->num_available == 0) ShowErrorMessage(STR_ERROR_NO_SONGS, INVALID_STRING_ID, WL_WARNING);
```

Comment:
STR_ERROR_NO_SONGS is also unused now, please also remove it in the "Cleanup" commit. (btw. "Cleanup" fits better than "Update" here)